### PR TITLE
Added support for AmazonAppstore. Updated sample to contain Amazon.

### PR DESCRIPTION
**Pull request by nmpsilva**
Just added support for the Amazon Appstore.
The default is Google Play, but added a new method to AppRate to change this to Amazon via an enum

### CODE REVIEW

File: library/src/main/java/hotchemi/android/rate/AppRate.java [hunk 1]
```diff
@@ -33,6 +33,11 @@
 
     private boolean isDebug = false;
 
+    public enum EAppstore {
```
Comment (hotchemi) [inline library/src/main/java/hotchemi/android/rate/AppRate.java]: I think it should be named just `Store`. Where E comes from?

File: library/src/main/java/hotchemi/android/rate/DialogOptions.java [hunk 2]
```diff
@@ -24,7 +28,8 @@
 
     private OnClickButtonListener listener;
 
-    public boolean shouldShowNeutralButton() {
+    public boolean shouldShowNeutralButton()
+    {
```
Comment (hotchemi) [inline library/src/main/java/hotchemi/android/rate/DialogOptions.java]: Could u follow the default formatter?

**Comment by hotchemi** [comment]
Nice idea. Thx:-)

File: library/src/main/java/hotchemi/android/rate/AppRate.java (hunk 1 above)
Comment (nmpsilva) [inline library/src/main/java/hotchemi/android/rate/AppRate.java]: The E stands for Enum, old-school habits. Will change it to ```StoreType```

**Comment by hotchemi** [comment]
@nmpsilva ping

**Comment by nmpsilva** [comment]
Sorry @hotchemi, this totally got lost on my queue of stuff to do.
Will submit another PR in no time.
My apologies once again.

**Pull request closed without merging**